packages/cli/src/cmds/init/initHandler.ts

```diff
@@ -23,7 +23,13 @@ import {
 } from './initUtils';
 
 /**
- * @param argv
+ * Creates a new snap package, based on one of the provided templates. This
+ * creates all the necessary files, like `package.json`, `snap.config.js`, etc.
+ * to start developing a snap.
+ *
+ * @param argv - The Yargs arguments object.
+ * @returns The Yargs arguments augmented with the new arguments.
```

Comment:
Perhaps we can elaborate on why there are new arguments 🤔 I'm not actually sure why

Reply:
I will let you figure out how to word it 😅